src/Config/Validator/Result/Error.php

```diff
@@ -52,6 +59,14 @@ public function getSuggestion(): string
         return $this->suggestion;
     }
 
+    /**
+     * @return integer|null
+     */
+    public function getErrorCode(): ?int
+    {
+        return $this->code;
```

Comment:
should not we use some default error code if it was not set?

Reply:
I don't know what default error code we can use in this case.

Reply:
if this method returns `null` we will have final error code as `1` ?

Reply:
Looks like yes